.github/mergeable.yml

```
@@ -0,0 +1,24 @@
+version: 2
+mergeable:
+  - when: pull_request.*
+    validate:
+      - do: title
+        must_exclude:
+          regex: ^WIP
+        must_include:
+          regex: ^((feature|chore|fix)(\(.{1,15}\))?|()\-[0-9]{1,5})\:.+$
```

How about following the types in the [contribution guide](https://github.com/monstar-lab-oss/admin-panel-template-reactjs/blob/main/CONTRIBUTING.md)?

```suggestion
          regex: ^((feat|fix|docs|style|refactor|perf|test|build|ci|chore)(\(.{1,15}\))?|()\-[0-9]{1,5})\:.+$
```